_api-reference/cluster-decommission.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Cluster decommission 
+nav_order: 20
+---
+
+# Cluster decommission
+
+The cluster decommission operation adds support for zonal cluster deployments based on awareness. When a particular zone overloads with traffic, you can decommission the zone to ensure uptime during large search requests. This greatly benefits use cases with multi-zone deployment, since rolling restarts per node can take more time than decommissioning a zone. 
+
```

Comment:
```suggestion
The cluster decommission operation adds support for decommissioning an entire awareness attribute(say zone). This greatly benefits use cases with multi-zone deployment, where zonal deployments can apply new upgrades in a controlled fashion. The cluster decommission also comes in handy when a particular zone's health becomes degraded during a zonal outage, in which case users can decommission that zone to prevent replication requests from getting stalled and request backlogs from building up. 
```

Reply:
Suggestion: change "comes in handy" to something like "is especially useful" or "can be very useful".

Reply:
say zone --> that is, zone?